<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeDialog</class>
 <widget class="QDialog" name="AttributeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttributeDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>显示文本标注</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>图形填充样式</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>纯颜色填充</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>右斜线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>左斜线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>斜网格线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>竖线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>横线</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>图形填充颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>填充图形内部空间</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>图形边框宽度设置（毫米）</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="currentText">
        <string>实线</string>
       </property>
       <item>
        <property name="text">
         <string>实线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>虚线</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>图形边框颜色设置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>图形边框线类型设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
